index.js

```diff
@@ -29,7 +31,11 @@ function externalLinks(options) {
       var data
       var props
 
-      if (ctx && isAbsoluteURL(ctx.url)) {
+      if (
+        ctx &&
+        isAbsoluteURL(ctx.url) &&
+        protocols.includes(ctx.url.slice(0, ctx.url.indexOf(':')))
```

Comment:
`indexOf` has better support across vendors, so I suggest using that. Maybe try a +1 or -1?
`f = 'https://example.com'; f.slice(0, f.indexOf(':'))` seems to work and return `https`, so it should work?